### PR TITLE
Enable TFM overrides per MultiTarget

### PR DESCRIPTION
**Pull request by Arlodotexe**
This PR adds conditions to the default declared TFMs for MultiTargets, enabling per-project TFM overrides without losing existing compatability with features that rely on MultiTarget declarations.

### CODE REVIEW

**Comment by Arlodotexe** [comment]
Needed by https://github.com/CommunityToolkit/Labs-Windows/pull/579

File: MultiTarget/AvailableTargetFrameworks.props [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
     <PropertyGroup>
-        <UwpTargetFramework>uap10.0.17763</UwpTargetFramework>
-        <WinAppSdkTargetFramework>net8.0-windows10.0.22621.0;net7.0-windows10.0.22621.0;net6.0-windows10.0.22621.0;</WinAppSdkTargetFramework>
+        <UwpTargetFramework Condition="'$(UwpTargetFramework)' == ''">uap10.0.17763</UwpTargetFramework>
+        <WinAppSdkTargetFramework Condition="'$(WinAppSdkTargetFramework)' == ''">net8.0-windows10.0.22621.0;net7.0-windows10.0.22621.0;net6.0-windows10.0.22621.0;</WinAppSdkTargetFramework>
         
-        <WasmHeadTargetFramework>net8.0</WasmHeadTargetFramework>
-        <LinuxHeadTargetFramework>net8.0</LinuxHeadTargetFramework>
-        <WpfHeadTargetFramework>net8.0</WpfHeadTargetFramework>
+        <WasmHeadTargetFramework Condition="'$(WasmHeadTargetFramework)' == ''">net8.0</WasmHeadTargetFramework>
```
Comment (Sergio0694) [inline MultiTarget/AvailableTargetFrameworks.props:6]: Not sure I understand the point of this:
- If these .props are imported before the .csproj, how are they ever going to see the .csproj overrides? And the .csproj being imported after them will still override the value anyway.
- If these .props are imported after the .csproj, why are they called .props and not .targets? 😅

Comment (Arlodotexe) [reply MultiTarget/AvailableTargetFrameworks.props:6]: They're imported inside the sample or source csproj, first through [ToolkitComponent.SourceProject.props](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/main/ToolkitComponent.SourceProject.props) or [ToolkitComponent.SourceProject.props](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/main/ToolkitComponent.SampleProject.props), which references [./MultiTarget/Library.props](https://github.com/CommunityToolkit/Tooling-Windows-Submodule/blob/main/MultiTarget/Library.props) to pick these up.

Comment (Sergio0694) [reply MultiTarget/AvailableTargetFrameworks.props:6]: Gotcha, ok so they're just called .props but they're effectively .targets, I get it now 😄

**Review by Sergio0694 — APPROVED** [review]

**Pull request merged**